FlowCrypt/Controllers/SignIn Other/EmailProviderViewController.swift

```diff
@@ -10,6 +10,11 @@ import AsyncDisplayKit
 import FlowCryptUI
 import Promises
 
+/**
+ * Controller that gives a possibility for the user to enter information about his email provider like, account, imap/smtp information
+ * - User redirects here from SignInViewController
+ * - After successful connection user will be redirected to *main flow*
+ */
 final class EmailProviderViewController: TableNodeViewController {
```

Comment:
We should call this `SetupImapViewController` ?

Reply:
Maybe better to call it `SetupEmailProviderViewController`? or if you think `SetupImapViewController` is better. naming - I'm ok with that as well

Reply:
This one is specifically for imap and smtp. I think email provider is too generic here. SetupImapViewController is preferred

Reply:
will change then in scope of #371